### PR TITLE
showfigfonts: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

I moved the `showfigfonts` to its own page.

### CODE REVIEW

**Comment by navarroaxel** [comment]
What do you think @victorhck ?

**Comment by victorhck** [comment]
Maybe can be added a 
> See also `showfigfont`

in figlet.md file.
Both are related

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/showfigfonts.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# showfigfonts
+
+> Display a list of available figlet fonts.
```
Comment (CleanMachine1) [inline pages/common/showfigfonts.md:3]:
```suggestion
> Display available figlet fonts.
```

File: pages/common/figlet.md [hunk 2]
```diff
@@ -9,12 +10,8 @@
 
 - Use a custom font file:
```
Comment (marchersimon) [inline pages/common/figlet.md:11]:
```suggestion
- Use a custom font file (full path  and extention can be omitted if the font file is in the default font directory):
```

File: pages/common/figlet.md [hunk 3]
```diff
@@ -9,12 +10,8 @@
...
 
-`figlet {{input_text}} -f {{font_filename}}`
+`figlet {{input_text}} -f {{path/to/font_file}}`
```
Comment (marchersimon) [inline pages/common/figlet.md:13]:
```suggestion
`figlet {{input_text}} -f {{path/to/font_file.flf}}`
```

**Review by marchersimon — COMMENTED** [review]
I'm not sure those changes are necessary, but maybe worth considering

File: pages/common/figlet.md (hunk 2 above)
Comment (navarroaxel) [reply pages/common/figlet.md:11]: @marchersimon I added a new example for this.

**Comment by marchersimon** [comment]
@navarroaxel shouldn't the new example also be in the Spanish translation?

**Comment by navarroaxel** [comment]
@marchersimon Yep, but I was expecting your approval.

**Review by marchersimon — APPROVED** [review]
Looks good, thanks 👍🏽

**Pull request merged**